app/src/lib/stats/stats-database.ts

```diff
@@ -187,6 +187,46 @@ export interface IDailyMeasures {
 
   /** The number of times the user opens the "Rebase current branch" menu item */
   readonly rebaseCurrentBranchMenuCount: number
+
+  /**
+   * The number of times the user has opened their external editor from the
+   * suggested next steps view
+   */
+  readonly suggestedStepOpenInExternalEditor: number
+
+  /**
+   * The number of times the user has opened their repository in Finder/Explorer
+   * from the suggested next steps view
+   */
+  readonly suggestedStepOpenWorkingDirectory: number
+
+  /**
+   * The number of times the user has opened their repository on GitHub from the
+   * suggested next steps view
+   */
+  readonly suggestedStepViewOnGitHub: number
+
+  /**
+   * The number of times the user has used the publish repository action from the
+   * suggested next steps view
+   */
+  readonly suggestedStepPublishRepository: number
+
+  /**
+   * The number of times the user has used the publish branch action branch from
+   * the suggested next steps view
+   */
+  readonly suggestedStepPublishBranch: number
+
+  /**
+   * The number of times the user has used the Create PR suggestion
+   * in the suggested next steps view. Note that this number is a
+   * subset of `createPullRequestCount`. I.e. if the Create PR suggestion
+   * is invoked both `suggestedStepCreatePR` and `createPullRequestCount`
+   * will increment whereas if a PR is created from the menu or from
+   * a keyboard shortcut only `createPullRequestCount` will increment.
+   */
+  readonly suggestedStepCreatePR: number
```

Comment:
:art: I'm not if we have strong feels for avoiding abbreviations, but this would help it align better with `createPullRequestCount`:

```suggestion
  readonly suggestedStepCreatePullRequest: number
```

But we do have some existing keys that don't follow this pattern:

 - `prBranchCheckouts`
 - `repoWithIndicatorClicked`
 - `repoWithoutIndicatorClicked`

FYIL I haven't made the changes to the other places where this is used as I wanted to flag it for discussion

Reply:
Aligned in 4149ad7ac